### DISCUSSION TITLE
can we please get a 2.4.10 release?

### DISCUSSION THREAD

**Discussion by dankamongmen** [onqtam/doctest#728] [General]
There have been multiple issues fixed in 2.4.9 for months, and these issues are causing various projects to not build in various distros. #665 would be an example. I know [Notcurses is broken](https://github.com/dankamongmen/notcurses/issues/2673) in Fedora Rawhide and other places. Would it be possible to get a 2.4.10 sometime soon? Debian hits freeze early next year.

Thanks! =]

**Comment by Saalvage** [comment]
I'm really sorry, but you know the thing, uni, job, life. I've mainly been struggling with grueling CI work for doctest, which I think I'll just skip for the time being to make way for 2.4.10.

**Comment by onqtam** [comment]
Hey, I just got v2.4.10 out of the door - a few clang builds fail (mostly because clang cannot be installed for some reason), but we decided that getting something out is better than nothing.

https://github.com/doctest/doctest/releases/tag/v2.4.10

**Reply by dankamongmen** [reply]
wonderful!
